### ISSUE TITLE
Accessibility - Add hidden option to hide title of card container

### ISSUE THREAD

**Issue by sandykadam** [visual-framework/vf-wp#1181]
Currently, we have card container on many pages but most of them doesn't have a title/heading which creates a problem for accessibility.

So we need to make the title mandatory in the card container and also add one more checkbox option to hide the title wherever needed this is to help screen readers show content in the hierarchy.  Even though if it is hidden selected we need to render the title but hide using `vf-u-sr-only` CSS class.

![Screenshot 2022-11-29 at 13 22 25](https://user-images.githubusercontent.com/1227241/204540134-d5d9411a-a13b-4dd6-a155-883a41410620.png)
![Screenshot 2022-11-29 at 13 22 31](https://user-images.githubusercontent.com/1227241/204540143-95109123-b3ae-49d0-93d7-0ee8a551c5f2.png)

cc: @nikiforosk

**Comment by kasprzyk-sz** [comment]
I updated the fields accordingly. The default for 'show header' will be 'Yes'.

![Screenshot 2022-12-01 at 13 17 34](https://user-images.githubusercontent.com/54020899/205050932-39676fd9-c1f6-4711-ae33-b57b6643f1ff.png)

**Comment by sandykadam** [comment]
Looks good and working thanks @kasprzyk-sz, closing this issue.
